src/igvfd/schemas/crispr_modification.json

```diff
@@ -122,6 +122,15 @@
                 "localizing",
                 "prime editing"
             ],
+            "enum_descriptions": {
+                "activation": "The CRISPR system is used to upregulate the expression of a target gene.",
+                "base editing": "The CRISPR system induces double-strand breaks at specific genomic locations to directly edit base sequences.",
+                "cutting": "Involves using a catalytically inactive Cas protein to bind to specific DNA sequences and block transcription.",
+                "interference": "CRISPR technique aimed at completely disabling a gene's function.",
+                "knockout": "Uses the CRISPR system to localize specific proteins or functional domains to particular genomic sites.",
```

Comment:
I don't think that's right at all; doesn't it actually cut the whole gene out?  Or am I confusing this with interference?